### DISCUSSION TITLE
How to move child to a different family?

### DISCUSSION THREAD

**Discussion by maco1717** [fisharebest/webtrees#4829] [General]
I have started adding a few kids. single parent kids. without realising. So it created every child on a separe family.

Is there any way to assign the same parent to all of them sowing them as the same family union?

**Comment by fisharebest** [comment]
I guess it depends exactly what you have done.

If you have a mother, with two children, each by different "unknown" fathers, and you want these to be the same (but unknown) father, then merge the two families.

If a child is simply in the wrong family, you go to the wrong family, and use the "change family members" to remove it.

Then either
* go to the new family, and use "change family members" to add the child
* go to the child, and use "link to existing family as a child"
